Gemfile

```diff
@@ -11,8 +11,11 @@ gem 'decidim-age_action_authorization', path: 'decidim-age_action_authorization'
 gem 'decidim-census', path: 'decidim-census'
 gem 'decidim-consultations'
 gem 'decidim-diba_census_api', path: 'decidim-diba_census_api'
+gem 'decidim-initiatives'
 gem 'decidim-ldap', path: 'decidim-ldap'
 
+gem "sprockets", "~> 3.7", "< 4"
+gem 'wicked_pdf'
```

Comment:
For sprockets and wicked_pdf comment why are we adding them.
Also for sprockets explain when we can get rid of this declaration.

Reply:
Done, changes applied.